packages/ods/src/components/select/src/components/ods-select/ods-select.scss

```diff
@@ -25,7 +25,7 @@ $select-padding-right: $select-caret-size + ($select-base-padding * 2);
     border: solid 1px var(--ods-color-neutral-300);
     border-radius: 0;
     background-color: $select-background-color;
-    cursor: pointer !important;
```

Comment:
still at least one use-case to handle then:
- open a select by clicking on the input => the cursor become a `text` one